frontend/src/views/applications/OperatingCostsView.vue

```diff
@@ -140,10 +164,54 @@ export default {
       } catch (error) {
         this.setFailureAlert('Failed to save your application', error)
       } finally {
+        this.processing = false
+        this.$emit('process', false)
+      }
+    },
+
+    async getDocuments() {
+      if (this.model.facilityType != FACILITY_TYPES.RENT_LEASE) return
```

Comment:
I'd be tempted to add a computed value isRentLease since it's used a fair bit.

Reply:
it is updated